clang/include/clang/3C/ProgramInfo.h

```diff
@@ -199,10 +203,6 @@ class ProgramInfo : public ProgramVariableAdder {
   // Retrieves a FVConstraint* from a Decl (which could be static, or global)
   FVConstraint *getFuncFVConstraint(FunctionDecl *FD, ASTContext *C);
 
-  // For each pointer type in the declaration of D, add a variable to the
-  // constraint system for that pointer type.
-  void addVariable(clang::DeclaratorDecl *D, clang::ASTContext *AstContext);
```

Comment:
Right, I forgot to ask about this. It looks like we have multiple interfaces, but only use one class. But this seems like a decent explanation so I'll follow your suggestion unless I hear otherwise.